tools/provision.sh

```diff
@@ -130,14 +130,12 @@ function platform_darwin_main() {
   brew_dependency osquery/osquery-local/zstd
 
   # List of LLVM-compiled dependencies.
-  brew_dependency osquery/osquery-local/lz4
   brew_dependency osquery/osquery-local/libmagic
   brew_dependency osquery/osquery-local/pcre
   brew_dependency osquery/osquery-local/boost
   brew_dependency osquery/osquery-local/asio
   brew_dependency osquery/osquery-local/cpp-netlib
   brew_dependency osquery/osquery-local/google-benchmark
-  brew_dependency osquery/osquery-local/snappy
```

Comment:
There should be a `brew_uninstall` bash function, use that, then bump the `rocksdb` revision again to uninstall from build hosts.